articles/Why-julia-is-slowly-replacing-python-for-machine-learning-and-data-science/index.md

```diff
@@ -0,0 +1,66 @@
+
+Julia is a worthy competitor for [Python](https://www.python.org/), particularly in the field of arithmetic coding. A preferred tool for developers for many years, Python is facing a threat of a modern programming language. Julia was developed to solve the challenges that Python has. Companies such as Facebook, Instagram, Spotify, Netflix, ILM, Dropbox, Yahoo!, Google and others use Python in one way or another.
+
+### Julia Language Merits
+
+Julia has many features and resources advantageous to machine-learning and data science. This language was designed with a focus on numerical and scientific computation. Julia&#39;s math-friendly syntax makes it ideal for users of [Matlab](https://www.mathworks.com/products/matlab.html), [Octave](https://www.gnu.org/software/octave/about), [Mathematica](https://www.mathematica.org/), [R](https://www.r-project.org/), among other computing languages and environments. With its own native machine learning libraries, Julia is expected to attract more data scientists in the future. An example of such a library is [Flux](https://justgetflux.com/) and is composed of several model patterns ideal for standard use cases. Flux is entirely written in Julia, meaning that users can implement modifications.
+
+### Python Language Advantages
+
+Unlike Julia, Python is a general-purpose language. Despite not being built purposely for data science, Python offers many advantages to machine-learning and data scientists. Python has existed for around 30 years in which it has established strong relationships with many third-party packages. This has attracted many users. One of the drawbacks associated with Python is speed. Better is that Python is implementing some great improvements, especially to the Python interpreter. Advances in parallel and multi-core processing are intended to make Python easier to speed up.
+
+### Tailored for Machine Learning
+
+Python is used for a broad range of tasks. Julia, on the other hand, is primarily developed to perform machine learning and statistics tasks. Because Julia is explicitly made for high-level statistical work, it has several benefits over Python. In linear algebra, for example, vanilla Julia shows better performance than [vanilla Python](https://pypi.org/project/vanilla/). This is mainly because, unlike Julia, Python does not support all equations and matrices performed in machine-learning. While Python is a great language, especially with [NumPy](https://numpy.org/), Julia beats it in when it comes to non-package experience with Julia feeling more catered towards machine learning calculations. The operand system of Julia can only be compared with that of R. That of Python is a bit weaker regarding performance, and that is a big setback.
```

Comment:
Wasn't quite sure what: "vanilla Julia" meant - or vanilla python - Im assuming it means (out the box) no refinements made.
Let be sure to clear this up for the audience. 🚀

Reply:
@EKahuha 
just noticing this was not address in the article